lib/ransack.rb

```diff
@@ -8,6 +8,12 @@
 module Ransack
   extend Configuration
   class UntraversableAssociationError < StandardError; end;
+
+  SUPPORTS_ATTRIBUTE_ALIAS = begin
+    ActiveRecord::Base.respond_to?(:attribute_aliases)
+  rescue NameError
+    false
+  end
 end
 
```

Comment:
`attribute_aliases` was introduced in Rails 4. So 3.x versions do not support it. I haven't found an alternative method in Rails 3.x that retrieves the aliases.